metals/src/main/scala/scala/meta/internal/process/SystemProcess.scala

```diff
@@ -53,8 +57,9 @@ object SystemProcess {
   def wrapProcess(
       ps: Process,
       redirectErrorOutput: Boolean,
-      processOut: String => Unit,
-      processErr: String => Unit
+      processOut: Option[String => Unit],
+      processErr: Option[String => Unit],
+      discardInput: Boolean
   ): SystemProcess = {
     def readOutput(stream: InputStream, f: String => Unit): Thread = {
       val filter = AnsiFilter()
```

Comment:
I really liked that you gave a thread a meaningful name before, could we do this now too?

Reply:
Good idea, yeah. Should be fixed now.